packages/features/insights/server/__tests__/getEventTypeList.test.ts

```diff
@@ -2,7 +2,7 @@ import { describe, expect, it, vi, beforeEach } from "vitest";
 
 import { readonlyPrisma } from "@calcom/prisma";
 
-import { getEventTypeList } from "../trpc-router";
+import { getEventTypeList } from "../getEventTypeList";
```

Comment:
Renamed this test to `getEventTypeList.test.ts` since the file's only about testing this function